contrib/inventory/ec2.py

```diff
@@ -399,7 +399,10 @@ def read_settings(self):
         if self.boto_profile:
             cache_dir = os.path.join(cache_dir, 'profile_' + self.boto_profile)
         if not os.path.exists(cache_dir):
```

Comment:
This check is now fairly pointless. If we're going to just `try`/`except`, no point checking it first.